kubernetes-maven-plugin/plugin/src/test/java/org/eclipse/jkube/maven/plugin/mojo/build/HelmPushMojoTest.java

```diff
@@ -169,6 +169,9 @@ void execute_withValidMavenSettings_shouldUpload() throws Exception {
       assertThat(helmPushMojo.jkubeServiceHub.getConfiguration().getPullRegistryConfig().getSettings()).singleElement()
           .isEqualTo(RegistryServerConfiguration.builder()
               .id("SNAP-REPO").username("mavenUser").password("mavenPassword").configuration(new HashMap<>()).build());
+      assertThat(helmPushMojo.jkubeServiceHub.getConfiguration().getPushRegistryConfig().getSettings()).singleElement()
```

Comment:
Why isn't assertion containing `getPullRegistryConfig()` removed?

Reply:
This test should be changed because it's testing multiple things at once.
I didn't want to spend more time on this yet because it's out of the scope of the current changes.
Basically, none of these assertions belong here, since what they are checking is that the Mojo performs the correct JKubeConfiguration initialization, which has nothing to do with "shouldUpload".
We need a (multiple) test(s) here and in other mojos that verify(ies) `shouldSetPushRegistry in JKubeConfiguration` and `shouldSetPullRegistry in JKubeConfiguration`.